.github/workflows/ci.yml

```diff
@@ -18,12 +18,10 @@ jobs:
         make all-via-docker
     - name: Install ckb-debugger
       run: |
-        cd ..
-        git clone https://github.com/nervosnetwork/ckb-standalone-debugger
-        cd ckb-standalone-debugger/ckb-debugger
-        cargo install --path .
+        cargo install --force --git https://github.com/nervosnetwork/ckb-standalone-debugger ckb-debugger
+        cargo install --force --git https://github.com/mohanson/ckb-standalone-debugger --branch syscall_spawn --root ~/.cargo-tmp/ ckb-debugger
```

Comment:
We don't need to install ckb-debugger with spawn in CI.
Just install it manually.